package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^8.2.1",
-    "husky": "^4.3.6",
+    "date-fns": "^2.16.1",
+    "firebase": "^8.2.4",
+    "husky": "^4.3.8",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
     "react-firebase-hooks": "^2.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.4"
+    "react-scripts": "^4.0.1"
```

Comment:
Hi! 

I'm so sorry I have a doubt, does it need to update other dependencies for your work or your new feature that you implemented? (`react`, `react-dom`, and `react-scripts`)

If it needs to update them, that's OK, I suggest that you can add more information or reasons about it in **"Testing Steps / QA Criteria"** section because we don't know why was motivation to change it. Maybe, we can learn something 😄 Please 🙏 , I'll appreciate it. (or Only update the `date-fns` dependency)

Reply:
Hi @alxmcr,

No, it was not necessary, only that the library that was installed (`date-nfs`) did not appear in the package files and it was necessary to upload them so that it did not appear error in this PR, we tried to update only this library (`date-fns`) but it still did not appear, so we had to update dependencies and there did appear the `date-nfs` library.

Thanks a lot for the comments and suggestions! They are very valuable for us. I will try again just to update that `date-fns` library. 😊

Reply:
@alxmcr It was not possible to update only that dependency (`date-fns`), the others are still being updated, so I added information as you suggested but in the changes section. This will not affect the functionality of the application.